gds/src/main/scala/edu/gemini/aspen/gds/configuration/KeywordConfigurationItem.scala

```diff
@@ -0,0 +1,53 @@
+package edu.gemini.aspen.gds.configuration
+
+import edu.gemini.aspen.giapi.data.ObservationEvent
+import edu.gemini.aspen.gds.fits._
+import edu.gemini.aspen.gds.model.KeywordSource
+
+case class Instrument(name: String)
+
+case class HeaderIndex(index: Int) {
+  require(index >= 0)
```

Comment:
I wonder if we should use refined types

Reply:
I thought about doing that, but decided to keep the number of dependencies down, esp. since we won't have help from scala steward. :( I can certainly use it if you would prefer that.